packages/babel-preset-default/bin/index.js

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+
+/**
+ * External dependencies
+ */
+const builder = require( 'core-js-builder' );
+const { minify } = require( 'uglify-js' );
+const { writeFile } = require( 'fs' ).promises;
+
+builder( {
+	modules: [ 'es', 'web' ],
+	// core-js is extremely conservative in which polyfills to include.
+	// Knowing about tiny browser implementation bugs that anyone rarely cares about,
+	// we prevent some features from having the full polyfill included.
+	// @see https://github.com/WordPress/gutenberg/pull/31279
+	exclude: [ 'es.promise' ],
+	targets: require( '@wordpress/browserslist-config' ),
+	filename: './build/polyfill.js',
+} )
+	.then( async ( code ) => {
+		const output = minify( code, {
+			output: {
+				comments: ( node, comment ) =>
+					comment.value.indexOf( 'License' ) > 0,
```

Comment:
Nitpick: I was expecting `>= 0` here, since `indexOf` resolves to `-1` if nothing is found.
`.includes()` would also be a good option :)

Reply:
Right, I assumed you will never start a comment with 'License` but I haven't checked the value so maybe it's better to stay on the safe side 👍🏻